src/controllers/user/me.rs

```diff
@@ -157,9 +157,13 @@ pub async fn update_user(
             // an invalid email set in their GitHub profile, and we should let them sign in even though
             // we're trying to silently use their invalid address during signup and can't send them an
             // email. They'll then have to provide a valid email address.
-            let _ = state
-                .emails
-                .send_user_confirm(user_email, &user.gh_login, &token);
+            let email = UserConfirmEmail {
+                user_name: &user.gh_login,
+                domain: &state.emails.domain,
+                token: &token,
+            };
+
+            let _ = state.emails.send(user_email, email);
```

Comment:
I realise this is exactly equivalent to the old code, but while we're in here, should we `?` on the result of `state.emails.send()` instead of swallowing it?

Reply:
one thing at a time 😉